R/utilities-grid.R

```diff
@@ -48,6 +48,17 @@ gg_par <- function(..., stroke = NULL, pointsize = NULL) {
   inject(gpar(!!!args))
 }
 
+#' The zero grob draws nothing and has zero size.
+#'
+#' @keywords internal
+#' @export
+zeroGrob <- function() .zeroGrob
+
+.zeroGrob <- NULL
+on_load(.zeroGrob <- add_class(nullGrob(), "zeroGrob"))
```

Comment:
The zeroGrob class is appended here, because `inherits(x, "zeroGrob")` is sometimes used instead of `is.zero()`. I expect the situation to be the same in extension packages.